lib/cloud/aws/policy_statements.go

```diff
@@ -57,6 +57,9 @@ func StatementForECSManageService() *Statement {
 			"ecs:DescribeServices", "ecs:CreateService", "ecs:UpdateService", "ecs:ListServices",
 			"ecs:RegisterTaskDefinition", "ecs:DescribeTaskDefinition", "ecs:DeregisterTaskDefinition",
 
+			// Required if the account has Resource Tagging Authorization enabled in Amazon ECS.
+			"ecs:TagResource",
```

Comment:
Is the `StatementForECSManageService` already deployed in prod?
If yes, do we need docs to let users know how to fix the problem?

Reply:
We don't have docs for this feature
I think we can try to catch this error in frontend and ask user to run the script again.
The script will have this new list of permissions and everything should work.

I've created an issue to track this.